packages/docs/site/docs/09-blueprints-api/05-steps.md

```diff
@@ -20,6 +20,132 @@ import BlueprintExample from '@site/src/components/Blueprints/BlueprintExample.m
 
 Each step is an object that contains a `step` property that specifies the type of step to run. The rest of the properties depend on the type of step. Learn and try each step type below.
 
+## Shorthands
+
+You can specify some `steps` using a `shorthand` syntax. The following `steps` are currently supported:
+
+### `login`
+
+Use
+
+```json
+	"login": true,
+```
+
+Or
+
+```json
+{
+	"step": "login",
+	"username": "admin",
+	"password": "password"
+}
+```
+
+### `plugins`
+
+(replaces `InstallPluginStep`)
```

Comment:
Perhaps this would look more familiar to the reader. `InstallPluginStep` is a TypeScript name but you never have to write it that way to use Blueprints:

```suggestion
(replaces the `installPlugin` step)
```